src/filters/H5Zblosc.jl

```diff
@@ -142,6 +135,15 @@ function BloscFilter(;level=5, shuffle=true, compressor="blosclz")
     BloscFilter(0,0,0,0,level,shuffle,compcode)
 end
 
+filterid(::Type{BloscFilter}) = H5Z_FILTER_BLOSC
+filtername(::Type{BloscFilter}) = blosc_name
+set_local_func(::Type{BloscFilter}) = blosc_set_local
+set_local_cfunc(::Type{BloscFilter}) = @cfunction(blosc_set_local, API.herr_t, (API.hid_t,API.hid_t,API.hid_t))
+filter_func(::Type{BloscFilter}) = blosc_filter
+filter_cfunc(::Type{BloscFilter}) = @cfunction(blosc_filter, Csize_t,
+                                                 (Cuint, Csize_t, Ptr{Cuint}, Csize_t,
+                                                 Ptr{Csize_t}, Ptr{Ptr{Cvoid}}))
```

Comment:
Is this still needed? Won't the generic one work?

Reply:
Yes, this is useful since all the arguments here are constants and `@cfunction` will return a pointer from than a closure `CFunction`. The closure depends on LLVM trampolines, which are not available on all platforms.

https://docs.julialang.org/en/v1/manual/calling-c-and-fortran-code/#Closure-cfunctions

Reply:
ah right. maybe add a comment to that effect?

Reply:
Added a comment to that effect in c450eba (still needs to be pushed to this branch)